web/cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
```

Comment:
I imagine we will need to add fixtures and intercepts for many of the portal features and to test edge cases. For instance the auth flow might require it.